assets/prototype/domain/shared/predicates/priceTypes/index.ts

```diff
@@ -0,0 +1 @@
+export * from './selectionPredicates';
```

Comment:
@tn3rb should `selectionPredicates` be exposed at this level ?

Reply:
i think so, the intent is for these functions to be the building blocks (ie: Lego) for more complex business logic.